_plugins/idp_translations.rb

```diff
@@ -0,0 +1,44 @@
+## This plugin copies the fonts and the images from the identity style guide
+## to the appropriate path on the built site
+##
+## The CSS and JS is already handled by Jekyll and the build:js npm script
+
+require 'fileutils'
+require 'uri'
+require 'net/http'
+require 'json'
+require 'open-uri'
+
+DOMAIN = Jekyll.configuration({})['idp_base_url']
+LANGUAGES = Jekyll.configuration({})['languages']
+MANIFEST_FILE = "/packs/manifest.json"
+
+def download_files(origin_uris, destination)
+  origin_uris.each do |uri|
+    stripped_name = uri.split('.').last(2).join('.')
+    FileUtils.mkdir_p(File.dirname("#{destination}/#{stripped_name}"))
+    download = URI.open("#{DOMAIN}#{uri}")
+    IO.copy_stream(download, "#{destination}/#{stripped_name}")
+  end
+end
+
+Jekyll::Hooks.register :site, :post_write do |site|
+  next if ([DOMAIN, LANGUAGES, MANIFEST_FILE].any?(&:nil?) | site.config['skip_idp_translations'])
```

Comment:
> Unfortunately, I had to access this value in the block's site param because Jekyll.configuration({}) didn't seem to pick up the alternative configuration 🤷

I would consider doing that for `DOMAIN` and `LANGUAGES` too so those variable overrides will work seamlessly.